applications/luci-app-watchcat/htdocs/luci-static/resources/view/watchcat.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+'require view';
+'require fs';
+'require form';
+'require tools.widgets as widgets';
+
+return view.extend({
+	load: function () {
+		return Promise.all([
+			L.resolveDefault(fs.stat('/usr/bin/watchcat.sh'), null)
+		]);
+	},
+	render: function (stats) {
```

Comment:
@nickberry17 thanks for the ping again. I have a lot to do at the moment and can't keep up with my work.

Why do we need **stats** here? I don't see you using that.
I therefore think the **load** function is superfluous and therefore also the **'require fs'** include.

You also have to do a rebase so that the translation fits again, otherwise I can't merge it.

Reply:
@feckert No pressure, I know how it can be sometimes.  I rebased, updated the ACL and refreshed the translation files and just ran a quick test on the latest master.  Are we ready to roll?